torch/csrc/jit/frontend/lexer.h

```diff
@@ -187,39 +187,40 @@ struct TORCH_API SharedParserData {
 #undef ADD_CASE
   }
 
-  // find the longest match of str.substring(pos) against a token, return true
-  // if successful filling in kind, start,and len
   bool match(
```

Comment:
@gmagogsfm Look here.

Reply:
Do you mean the perf optimization is here? I don't see one. Could you elaborate?